examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1672 @@
+{
```

Comment:
<p>Q:Why is this done?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
In order to ensure that models are initialized with constant weights for reproducible results.